spec/features/single_content_item_spec.rb

```diff
@@ -1,12 +1,12 @@
 RSpec.describe '/metrics/base/path', type: :feature do
   include GdsApi::TestHelpers::ContentDataApi
-
+  let(:metrics) { MetricsService::DEFAULT_METRICS }
```

Comment:
I would rather have explicit fixtures and be explicit setting up the fixtures (do not rely on production code). 
In this case, we can be specific on line 3 with the metrics that we will be asking for, and then, in the application code we just use a constant.
The reason behind this is: if we decide to change the way we store the metrics (not use a constant anymore), then I don't have to change the test because it would be risky.